ui/src/app/subnets/views/VLANDetails/VLANDetails.tsx

```diff
@@ -77,7 +84,10 @@ const VLANDetails = (): JSX.Element => {
       <DHCPStatus />
       <ReservedRanges />
       <VLANSubnets />
-      <DHCPSnippets />
+      <DHCPSnippets
+        subnetIds={subnets.map(({ id }) => id)}
```

Comment:
There's no list of subnet IDs on the vlan object? That's lame. Maybe we should request an update to the websocket API?

Reply:
Good idea, I filed: https://bugs.launchpad.net/maas/+bug/1959648.